contracts/oracles/BinanceOracle.sol

```diff
@@ -7,8 +7,9 @@ import "../interfaces/VBep20Interface.sol";
 import "../interfaces/SIDRegistryInterface.sol";
 import "../interfaces/FeedRegistryInterface.sol";
 import "../interfaces/PublicResolverInterface.sol";
+import "@venusprotocol/governance-contracts/contracts/Governance/AccessControlledV8.sol";
 
-contract BinanceOracle is Initializable {
+contract BinanceOracle is Initializable, AccessControlledV8 {
```

Comment:
I think we can remove redundant `Initializable` it is already imported in `AccessControlledV8`